### DISCUSSION TITLE
Microsoft Teams Notifier: Reminder Alerts

### DISCUSSION THREAD

**Discussion by calvinmo** [grafana/grafana#51479] [Alerting]
There are a number of native notifiers built into Grafana which support periodic reminder alerts (eg: Slack).  This was very useful to us.  We have since moved across to Microsoft Teams for our chosen Instant messaging system and unfortunately the Teams notifier does not offer this same functionality.

Is it possible to add this?  The user interface would probably be very similar to that of the Slack notifier.  An optional reminder tickbox with a editable frequency defaulting to 15 minutes.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
